spot-client/src/common/app-state/spot-tv/actions.js

```diff
@@ -49,13 +49,11 @@ export function setSpotTVState(newState) {
  *
  * @param {Array<{ type: string, number: string }>} [invites] - An array of invites as specified by Jitsi Meet iframe
  * API.
- * @returns {Function}
+ * @returns {Object}
  */
 export function storePhoneNumberFromInvites(invites) {
-    return dispatch => {
-        const phone = invites?.filter(invite => invite.type === 'phone')?.[0]?.number;
+    const phone = invites?.find(invite => invite.type === 'phone')?.number;
```

Comment:
style nit: maybe a default arg for invites? That way it's always defined.

Reply:
yeah, but too lazy to change it given `.?` is now so easy